src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> getDataFromInputFile(String fileName) {
+        File inputDataFile = new File("src/main/resources/" + fileName);
+        try {
+            inputDataFile.createNewFile();
```

Comment:
why are you creating file here? File reader has to only read existing file

Reply:
not fixed